integration/options.go

```diff
@@ -54,8 +54,8 @@ const (
 	// defaultWorkerPoolSize is the default number of workers in the worker pool.
 	defaultWorkerPoolSize = 10
 
-	// defaultTickInterval is the default tick interval.
-	defaultTickInterval = 1 * time.Second
+	// defaultTickMinimumInterval is the default tick interval.
```

Comment:
nit: is the default minimum tick interval

Reply:
Good catch, will updated.